docs/calendar-draft.md

```diff
@@ -0,0 +1,315 @@
+	[Temporal.Calendar.plus](
+		self: Temporal.Date,
+		duration: Temporal.Duration,
+		options: /* options bag */
+	) : Temporal.Date;
+
+	/** Returns self minus duration according to the calendar rules. */
+	[Temporal.Calendar.minus](
+		self: Temporal.Date,
+		duration: Temporal.Duration,
+		options: /* options bag */
+	) : Temporal.Date;
+
+	/** Returns self minus other, which are dates in the same calendar. */
+	[Temporal.Calendar.difference](
+		self: Temporal.Date,
+		other: Temporal.Date,
+		options: /* options bag */
+	) : Temporal.Duration;
+
+	////////////////////////////////////
+	//  Accessors:                    //
+	//  Semantics defined in date.md  //
+	////////////////////////////////////
+
+	[Temporal.Calendar.dayOfWeek](
+		self: Temporal.Date
+	) : number;
+
+	[Temporal.Calendar.weekOfYear](
+		self: Temporal.Date
+	) : number;
+
+	[Temporal.Calendar.daysInMonth](
+		self: Temporal.Date
+	) : number;
+
+	[Temporal.Calendar.daysInYear](
+		self: Temporal.Date
+	) : number;
+
+	[Temporal.Calendar.leapYear](
+		self: Temporal.Date
+	) : number;
+}
+```
+
+An instance of `MyCalendar` is expected to have stateless behavior; i.e., calling a method with the same arguments should return the same result each time.
+
+## Default Calendar
+
+An open question is what the behavior should be if the programmer does not specify a calendar, or if we should require the programmer to always specify a calendar.  Four choices are on the table:
+
+1. Default to full ISO (Gregorian) calendar.
+2. Require the user to explicitly specify the calendar.
+3. Default to a partial ISO calendar (explained below).
+4. Default to `Intl.defaultCalendar` (a new symbol), or ISO if that field doesn't exist.
+
+### Partial ISO Calendar (Option 3)
+
+A partial ISO calendar would be one implemented as follows:
+
+```javascript
+const PartialIsoCalendar = {
+	[Temporal.Calendar.toISO] = (self) => {
+		return self;
+	},
+
+	[Temporal.Calendar.fromISO] = (isoDate) => {
+		return isoDate;
+	},
+
+	[Temporal.Calendar.id] = "iso",
+
+	// ALL OTHER METHODS:
+	[Temporal.Calendar.plus] = () => {
+		throw new TypeError("Unsupported operation: full calendar required");
+	}
+	// Same for [Temporal.Calendar.minus], etc.
+}
+```
+
+It would in effect render default Temporal.Date (and Temporal.DateTime) with fewer operations until you specify a calendar.  The following methods/getters would throw:
+
+- .dayOfWeek
+- .weekOfYear
+- .daysInMonth
+- .daysInYear
+- .leapYear
+- .plus() -- *might* be OK if the programmer requests only time units
+- .minus() -- *might* be OK if the programmer requests only time units
+- .difference() -- *might* be OK if the programmer requests only time units
+- .getYearMonth()
+- .getMonthDay()
+
+The following methods/getters would still work:
+
+- .with()
+- .withTime()
+- .toString()
+- .toLocaleString()
+- .compare()
+
+Although small, this set of operations still covers many of the recipes in the Temporal Cookbook.
```

Comment:
The recipes theirselves do not exist, so this claim seems a little too strong. I think the Temporal proposal continues to let those linger to its detriment (since they are intended to _inform_ the API design), but that's a topic for another issue.

Reply:
I'm referring to #240 as the cookbook, which has a list of 18 recipes.  I'll change the language to "proposed cookbook".  Resolving this issue because I don't know how else it is actionable.